src/client/api/container.c

```diff
@@ -142,44 +142,12 @@ daos_cont_create_with_label(daos_handle_t poh, const char *label,
 	return rc;
 }
 
```

Comment:
Isn't this a major API change breaking backward compatibility?

API_VERSION_MAJOR should be updated in SConstruct.

Reply:
Yes. We are removing many compatibility code. Only one form, int daos_cont_open(daos_handle_t poh, const char *cont, unsigned int flags, daos_handle_t *coh, daos_cont_info_t *info, daos_event_t *ev), is kept. Johann suggested to use *2 suffix. I guess our API_VERSION_MAJOR is still 2. I am checking with Johann to confirm.